src/classes/RP_GitHubController.cls

```diff
@@ -0,0 +1,131 @@
+/**
+* @author Salesforce.org
+* @date 2017
+* @group Getting Started
+* @description Component that brings the latest release notes from Github using Salesforce org HEROKU APP.
+*/
+public with sharing class RP_GitHubController {
+
+    //Object used to load all information related to github release notes
+    public GitHubWrapper gitHubData {get; set;}
+    public Boolean timeoutError {get; set;}
+
+    public RP_GitHubController() {
+
+        loadLatestReleaseNotes();
+    }
+
+    /*******************************************************************************************************
+    * @description method to load release notes from github, into GithubWrapper object and use it in github component
+    * @return void
+    */
+    @TestVisible private void loadLatestReleaseNotes() {
+
+        RP_GitHubClient client = new RP_GitHubClient();
+        HttpResponse response = client.getLatestReleaseNotes();
+        timeoutError = true;
+
+        if (response != null) {
+
+            String responseBody = response.getBody();
+
+            Map<String, Object> responseBodyMap = (Map<String, Object>) JSON.deserializeUntyped(responseBody);
+
+            gitHubData = new GitHubWrapper(responseBodyMap);
+
+            timeoutError = false;
+        }
+
+    }
+
+    /*******************************************************************************************************
+    * @description inner class, used to access all github release notes data
+    */
+    public class GitHubWrapper {
+
+        public String latestDate {get; set;}
+        public String latestDownloadUrl {get; set;}
+        public Object latestVersion {get; set;}
+        public String latestReleaseNotes {get; set;}
+        public Object latestProductName {get; set;}
+
+
+        /*******************************************************************************************************
+        * @description GithubWrapper constructor, used to set response map from salesforce org endpoint, into
+        * a GithubWrapper object
+        * @param a deserialize json load in a map with the response values from salesforce org api
+        */
+        public GitHubWrapper(Map<String, Object> responseBodyMap) {
+
+            if (responseBodyMap != null && responseBodyMap.containsKey('production_release')) {
+
+                Map<String, Object> latestReleaseMap = (Map<String, Object>) responseBodyMap.get('production_release');
+                latestProductName = responseBodyMap.get('product_name');
+
+                if (latestReleaseMap != null && !latestReleaseMap.isEmpty()) {
+
+                    latestReleaseNotes = (String) latestReleaseMap.get('release_notes_html');
+                    latestDownloadUrl = (String) latestReleaseMap.get('url');
+                    latestVersion = (String) latestReleaseMap.get('version');
+
+                    String timeCreated = (String) latestReleaseMap.get('time_created');
+                    latestDate = formatDate(timeCreated);
```

Comment:
@cpurodriguez here you are using time_created, but I think you should use time_push_prod